lib/workers/repository/update/branch/handle-existing.ts

```diff
@@ -1,13 +1,31 @@
 import { GlobalConfig } from '../../../../config/global';
 import { logger } from '../../../../logger';
 import type { Pr } from '../../../../modules/platform';
-import { ensureComment } from '../../../../modules/platform/comment';
+import {
+  ensureComment,
+  ensureCommentRemoval,
+} from '../../../../modules/platform/comment';
 import { PrState } from '../../../../types';
+import { emojify } from '../../../../util/emoji';
 import { branchExists, deleteBranch } from '../../../../util/git';
 import * as template from '../../../../util/template';
 import type { BranchConfig } from '../../../types';
 
 export async function handlepr(config: BranchConfig, pr: Pr): Promise<void> {
```

Comment:
```suggestion
export async function handleModifiedPr(config: BranchConfig, pr: Pr): Promise<void> {
```

While we're changing things, let's try to make the file names and function names be a bit more explanatory.

Is this name more accurate? And also is `handle-existing.ts` accurate? Seems we only call this if a branch is modified.

Reply:
We are also calling it for closed/merged PRs 
overall this handles these `PrState.[Open|Closed|Merged]`.

https://github.com/renovatebot/renovate/blob/d869c946d164965393f1259db5171e2a0303fb27/lib/workers/repository/update/branch/index.ts#L118-L123

`handlePrComments` maybe (?)

Reply:
Should we call the same function in both cases if the logic is entirely separate?

Reply:
not really, ill drop the wrapping function